pkg/config/diagnostics/config.go

```diff
@@ -1,8 +1,14 @@
 package diagnostics
 
-import "github.com/kumahq/kuma/pkg/config"
+import (
+	"github.com/pkg/errors"
+
+	"github.com/kumahq/kuma/pkg/config"
+)
 
 type DiagnosticsConfig struct {
+	// Port of Diagnostic Server for checking health and readiness of the Control Plane
+	ServerPort int `yaml:"serverPort" envconfig:"kuma_diagnostics_server_port"`
```

Comment:
make it uint32 and skip the validation (or do the validation that it's lower than 65k)